src/main/java/org/terasology/fluid/system/FluidManager.java

```diff
@@ -17,71 +17,74 @@
 
 import org.terasology.entitySystem.entity.EntityRef;
 
+/**
+ * Interface for a generic fluid manager.
+ */
 public interface FluidManager {
     /**
      * Add a certain volume of fluid to all fluid inventory slots.
      *
-     * @param instigator    The entity that's instigating this action.
```

Comment:
As mentioned above, this is actually the correct way to do parameter javadocs. Undo this change, please 🙂